sale_stock_available_to_promise_release/i18n/fr_CH.po

```diff
@@ -6,7 +6,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 13.0+e\n"
```

Comment:
I really think this file should be renamed fr.po so it is useful for every French speaking localisation.
@mmequignon

Reply:
Done